src/components/CardSchemes/CardSchemes.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
+
+import styled, { css } from 'react-emotion';
+import PaymentMethodIcon from './components/PaymentMethodIcon';
+import { sizes } from '../../styles/constants';
+
+const { BYTE, KILO, MEGA, GIGA } = sizes;
+
+const CardSchemeBaseStyles = ({ theme }) => css`
+  box-sizing: content-box;
+  display: inline-block;
+  padding: ${theme.spacings.byte};
+`;
+
+const listWrapperBaseStyles = () => css`
+  display: block;
+  line-height: 0;
+  text-align: center;
+`;
+
+const CardSchemeWrapper = styled('ul')(listWrapperBaseStyles);
+const CardScheme = styled('li')(CardSchemeBaseStyles);
+
+/**
+ *   Displays a row of available or active card scheme icons
+ */
+const CardSchemes = ({ schemeIds, size }) => {
+  if (isEmpty(schemeIds)) {
+    return null;
+  }
+
+  return (
+    <CardSchemeWrapper>
+      {schemeIds.map(schemeId => (
+        <CardScheme key={schemeId}>
+          <PaymentMethodIcon size={size} schemeId={schemeId} />
```

Comment:
Now that this is a `PaymentMethodIcon`, why is the prop still called `schemeId`? Shouldn't it be `iconId` or something more general than a scheme?